Projects/Feature/InputProfileInfoFeature/Sources/Scene/InputProfileInfoView.swift

```diff
@@ -7,6 +8,94 @@ struct InputProfileInfoView: View {
+
+                VStack(spacing: 32) {
+                    HStack(spacing: 4) {
+                        Text("프로필")
+                            .foregroundColor(.sms(.system(.black)))
+
+                        Text("*")
+                            .foregroundColor(.sms(.sub(.s2)))
+
+                        Spacer()
+
+                        SMSPageControl(pageCount: 6, selectedPage: 0)
+                    }
+                    .smsFont(.title1)
+
+                    VStack(alignment: .leading, spacing: 24) {
+                        ZStack(alignment: .bottomTrailing) {
+                            SMSIcon(.profile, width: 100, height: 100)
+
+                            SMSIcon(.profileSmallPlus)
+                                .overlay {
+                                    RoundedRectangle(cornerRadius: 7)
+                                        .strokeBorder(Color.sms(.system(.white)), lineWidth: 4)
+                                }
+                                .offset(x: 5, y: 4)
+                        }
+                        .titleWrapper("사진")
+
+                        SMSTextField(
+                            "1줄 자기소개입력",
+                            text: Binding(get: { state.introduce }, set: intent.updateIntroduce(introduce:))
+                        )
+                        .titleWrapper("자기소개")
+
+                        SMSTextField(
+                            "공개용 이메일입력",
+                            text: Binding(get: { state.email }, set: intent.updateEmail(email:))
+                        )
+                        .titleWrapper("이메일")
+
+                        SMSTextField(
+                            "Frontend",
+                            text: Binding(get: { state.major }, set: intent.updateMajor(major:))
+                        )
+                        .disabled(true)
+                        .overlay(alignment: .trailing) {
+                            SMSIcon(.downChevron)
+                                .padding(.trailing, 12)
+                        }
+                        .titleWrapper("분야")
+                        .onTapGesture {
+                            intent.majorSheetIsRequired()
+                        }
+
+                        SMSTextField(
+                            "https",
+                            text: Binding(get: { state.portfoiloURL }, set: intent.updatePortfoiloURL(portfoiloURL:))
+                        )
+                        .titleWrapper("포트폴리오 URL")
+
+                        SMSTextField(
+                            "예시) Swift, UIKit, SwiftUI",
+                            text: Binding(get: { state.techStack }, set: intent.updateTechStack(techStack:))
+                        )
+                        .titleWrapper("세부스택")
+                    }
+
+                    CTAButton(text: "다음")
+                        .padding(.bottom, 32)
+                }
+                .padding(.top, 20)
+                .padding(.horizontal, 20)
```

Comment:
```suggestion
                .padding([.top, .horizontal], 20)
```
둘이 같은 값의 padding이면 이렇게 처리 해주는건 어떨가요?

Reply:
좋네용